doc/docs/usage/upgrading/v0_2_to_v0_3.md

```diff
@@ -0,0 +1,26 @@
+# Upgrading v0.2 to v0.3
+
+When upgrading from v0.2 to v0.3, the installation script simply needs to be run again. However, due to breaking changes, some manual adjustments are required for existing configurations and workloads.
+
+## Configurations
+
+`CompleteState`
+
+* `currentState` has been renamed to `desiredState`
+
+`State`
+
+* New mandatory field `apiVersion`
+* The field `restart` has been renamed to `restartPolicy` and the value is not boolean but an enum.
+
+## API
+
+Application using the control interface or communication directly with the Ankaios server need to be adapted.
+
+A new `ToServer` message, `Request`, has been introduced.
```

Comment:
Is the `ToServer` new message? Or just a renaming of `StateChangeRequest`?

Reply:
I think it was meant as message type.

Reply:
I have now also described the renames and clarified the sentences about the new types, Response and Request.